src/components/number-picker/number-picker.component.ts

```diff
@@ -117,6 +118,12 @@ export class NumberPickerComponent implements ControlValueAccessor, OnDestroy {
         this._isDestroyed = true;
     }
 
+    ngOnChanges(changes: SimpleChanges): void {
+        if (changes.valid && changes.valid.isFirstChange()) {
+            console.warn(`Number picker [valid] property has been deprecated. Instead use reactive form validation.`);
```

Comment:
We should display the component name since this could potentially be ambigious in a console log, i.e. "ux-number-picker [valid] property has been deprecated..."

Reply:
updated now